### PR TITLE
Jack

### PR DESCRIPTION
**Pull request by JackyWang96**
Update test73588

### CODE REVIEW

**Comment by JackyWang96** [comment]
Q: Why cy.get('#randomtext').should('not.be.empty') 会报错

File: cypress/e2e/test73588.cy.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
+describe('DropDown Select Test', ()=>{
+   it('select combobox', ()=> {
```
Comment (jun1983) [inline cypress/e2e/test73588.cy.ts:4]: Please name the 'it' with the problem id so it is easy to understand what problem it is solving.

File: cypress/e2e/test73588.cy.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                       cy.get('#selectlink').select($el.text());
+                   }
+               }
+           );
+           cy.get('#submitanswer').click();
+       });
+   });
+});
+
+
```
Comment (jun1983) [inline cypress/e2e/test73588.cy.ts:26]: After all changes, you should finally write code as below:

it('73588', ()=> {
      cy.visit('https://obstaclecourse.tricentis.com/Obstacles/73588/')
      cy.get('#clickme').click()
      cy.get('#randomtext').should('not.have.value', '')
      cy.get('#randomtext').invoke('val').then((val)=>{
        cy.get('#selectlink').select(val)
      })
      cy.get('#submitanswer').click()
      cy.contains("Good job!")
    })

**Comment by jun1983** [comment]
> Q: Why cy.get('#randomtext').should('not.be.empty') 会报错

Because the element is an input, not text field. You were checking text is not empty. Instead, you should check the value.

File: cypress/e2e/test73588.cy.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
```
[the remark this answers is not in the document]
Comment (JackyWang96) [reply cypress/e2e/test73588.cy.ts:1]: Should we add question no. as well? e.g test73588ByJack.cy.ts?

[the remark this answers is not in the document]
Comment (jun1983) [reply cypress/e2e/test73588.cy.ts:1]: Not required. A test file can have multiple test cases.

**Pull request closed without merging**